GPyOpt/testing/core_tests/test_save.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import numpy as np
+from GPyOpt.methods import BayesianOptimization
+from GPyOpt.models import GPModel
+import GPy
+import tempfile
+import os
+
+
+class TestSaveModel(unittest.TestCase):
```

Comment:
Would you mind expanding the tests a little bit here? Even if it is simply looking for more than just "Iterations" - it could also look for specific numbers (with some short precision, say two digits) or check the number of rows